packages/mui-base/src/Menu/Item/useMenuItem.ts

```diff
@@ -36,6 +38,11 @@ export function useMenuItem(params: useMenuItem.Parameters): useMenuItem.ReturnV
           id,
           role: 'menuitem',
           tabIndex: highlighted ? 0 : -1,
+          onKeyUp: (event: React.KeyboardEvent & MuiCancellableEvent) => {
```

Comment:
I'm going to refactor `useButton` to use the new method of preventing our logic in a separate PR